crates/python/pyproject.toml

```diff
@@ -1,17 +1,43 @@
+# This is the metadata Maturin uploads to PyPI on publish
+[project]
+name = "qcs-sdk-python"
+version = "0.1.0"
+description = "Python interface for the QCS Rust SDK"
+readme = "README.md"
+license = { text = "Apache-2.0" }
+authors = [
+	{ name = "Rigetti Computing", email = "softapps@rigetti.com" },
+	{ name = "Mark Skilbeck", email = "mark.skilbeck@rigetti.com" },
+	{ name = "Marquess Valdez", email = "mvaldez@rigetti.com" },
+	{ name = "Randall Fulton", email = "rfulton@rigetti.com" },
+]
+classifiers = [
+	"Development Status :: 3 - Alpha",
+	"License :: OSI Approved :: Apache Software License",
+	"Programming Language :: Python :: 3.7",
+	"Programming Language :: Python :: 3.8",
+	"Programming Language :: Python :: 3.9",
+	"Operating System :: OS Independent",
+]
+
+# PEP 621 specifies the [project] table as the source for project metadata. However, Poetry only supports [tool.poetry]
+# We can remove this table once this issue is resolved: https://github.com/python-poetry/poetry/issues/3332
 [tool.poetry]
 name = "qcs-sdk-python"
-version = "0.1.0-rc1"
+version = "0.1.0"
 description = "Python interface for the QCS Rust SDK"
-license = "Apache-2.0"
+readme = "README.md"
```

Comment:
Did you intend to remove the `license` item?

Reply:
Correct, `maturin upload` uses the metadata in the `[project]` table. The metadata in `[tools.poetry]` is just what `poetry` needs to not complain. I removed everything else to minimize duplication.